server/src/main/scala/slickdb/SeminarsRegistration.scala

```diff
@@ -0,0 +1,52 @@
+package slickdb
+
+import drt.shared.{SeminarRegistration}
+import org.joda.time.DateTime
+import slick.lifted.ProvenShape
+import slick.jdbc.PostgresProfile.api._
+
+import java.sql.Timestamp
+import scala.concurrent.{ExecutionContext, Future}
+
+case class SeminarsRegistrationRow(email: String,
+                                   seminar_id: Int,
+                                   register_time: Timestamp,
+                                   email_sent: Option[Timestamp]) {
+  def toSeminarRegistration: SeminarRegistration = SeminarRegistration(email, seminar_id, register_time.getTime, email_sent.map(_.getTime))
+}
+
+class SeminarsRegistration(tag: Tag) extends Table[SeminarsRegistrationRow](tag, "seminar_registration") {
+
+  def email: Rep[String] = column[String]("email")
+
+  def seminarId: Rep[Int] = column[Int]("seminar_id")
+
+  def registerTime: Rep[Timestamp] = column[Timestamp]("register_time")
```

Comment:
It's quite nice to use `..At` for timestamp columns so it's really clear what it's used for. So we can have:
`def registeredAt: Rep[Timestamp] = column[Timestamp]("registered_at")`
`def emailSentAt: Rep[Option[Timestamp]] = column[Option[Timestamp]]("email_sent_at")`